packages/amplify-util-uibuilder/src/utils/prePushHandler.ts

```diff
@@ -1,10 +1,14 @@
 import { $TSContext } from 'amplify-cli-core';
-import type { Schema } from '@aws-amplify/datastore';
 import { printer } from 'amplify-prompts';
 import AmplifyUIBuilder from 'aws-sdk/clients/amplifyuibuilder';
 import { AmplifyStudioClient } from '../clients';
 import { isFormDetachedFromModel, isFormSchemaCustomized, shouldRenderComponents } from '../commands/utils';
 
+// TODO: replace this type when amplify-codegen package export official types
```

Comment:
I would suggest to create an asana task blocked on amplify-codegen export types task, otherwise TODOs tend to remain orphaned in the code

Reply:
asana task created. thanks